elastic/shared/runners/remote_cluster.py

```diff
@@ -1,143 +1,183 @@
+
+        for remote_cluster_name, remote_cluster_client in remaining_es_clients.items():
+            # retrieve seed nodes from the remote cluster
+            self.logger.info(
+                f"retrieving nodes from the remote cluster [{remote_cluster_name}]",
+            )
+            remote_nodes_resp = await remote_cluster_client.nodes.info()
+            remote_seed_nodes = self._get_seed_nodes(remote_nodes_resp)
+            # ensure the identifier used as the 'remote' name matches the "remote*" pattern for use in query tasks
+            remote_cluster_identifier = f"remote_{remote_cluster_name}"
+
+            # connect 'local' cluster to 'remote' cluster
+            await self._configure_remote_cluster(
+                local_cluster_client=local_es,
+                local_cluster_name=local_cluster_name,
+                remote_cluster_identifier=remote_cluster_identifier,
+                remote_seed_nodes=remote_seed_nodes,
+            )
+
+            # connect 'remote' cluster to 'local' cluster
+            await self._configure_remote_cluster(
+                local_cluster_client=remote_cluster_client,
+                local_cluster_name=remote_cluster_name,
+                remote_cluster_identifier=local_cluster_name,
+                remote_seed_nodes=local_seed_nodes,
+            )
+
     def __repr__(self, *args, **kwargs):
-        return "configure-remote-cluster"
+        return "configure-remote-clusters"
 
 
-class FollowIndexRunner(Runner):
+class ConfigureCrossClusterReplication(Runner):
     """
-    Runner to replicate and follow indices from the remote cluster to the local cluster using CCR.
-    * `local-cluster`: (mandatory) the name of the local cluster specified in `target-hosts` parameters.
-    * `remote-cluster`: (mandatory) the name of the remote cluster specified in `target-hosts` parameters
+    Runner that configures all other clusters specified in `target-hosts` to follow specified indices from the 'source-cluster'
+    * `source-cluster`: (mandatory) the name of the cluster specified in `target-hosts` parameters from which to follow indices.
     * `index`: (mandatory) the pattern of indices to be replicated
     * `request-timeout`: (optional) the timeout of this runner. Defaults to 7200
-    * `remote-connection-name`: (optional) the name of the remote connection;
-    *                           defaults to the name of the remote cluster
     """
 
     multi_cluster = True
 
     def __init__(self):
         super().__init__()
 
-    async def __call__(self, multi_es, params):
-        end_request_timeout = time.process_time() + params.get("request-timeout", 7200)
-        required_licenses = ["trial", "platinum", "enterprise"]
-
-        def request_timeout():
-            return math.ceil(end_request_timeout - time.process_time())
-
-        remote_es = multi_es[params["remote-cluster"]]
-        local_es = multi_es[params["local-cluster"]]
+    def request_timeout(self, timeout):
```

Comment:
do we need this function? It seems like it always returns `timeout` unless `time.process_time()` takes > 1s to run

Reply:
To be honest this was copied over from the original implementation, I agree it doesn't make sense to keep. Simplified in https://github.com/elastic/rally-tracks/pull/333/commits/9263e4a726ad28c47fdf52d253fa165f300403a7.

Reply:
OK. I was gonna say, if we need the runner to be slower or more complicated I can come up with some pretty funny stuff to do